composer.json

```diff
@@ -0,0 +1,14 @@
+{
+    "name": "cn-consult/pbshtool",
+    "description": "ProfitBricks Snapshot Tool",
+    "type": "library",
+    "require": {
+        "symfony/console": "^3.1"
```

Comment:
I did not check exact which versions of symfony console commands are compatible and which are not.